### PR TITLE
GitHub: Don't trigger source code checks

### PR DESCRIPTION
**Pull request by JonatanAntoni**
... when only documentation is touched

### CODE REVIEW

File: .github/workflows/buildmgr.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ on:
       - 'libs/rtemodel/**'
       - 'libs/rtefsutils/**'
       - 'tools/buildmgr/**'
+      - '!**/*.md'
```
Comment (soumeh01) [inline .github/workflows/buildmgr.yml:17]: Though it doesn't make much difference, would it be better to separate the include and exclude paths?
```
on:
  push:
    paths-ignore:
      - **/*.md
```

Comment (JonatanAntoni) [reply .github/workflows/buildmgr.yml:17]: Afaik, `paths` and `paths-ignore` are mutually-exclusive.

Comment (soumeh01) [reply .github/workflows/buildmgr.yml:17]: Ahh I see, 

`You can not use paths and paths-ignore to filter the same event in a single workflow. If you want to both include and exclude path patterns for a single event, use the paths filter along with the ! character to indicate which paths should be excluded.`

**Review by soumeh01 — APPROVED** [review]
LGTM

**Review by soumeh01 — APPROVED** [review]

**Review by jkrech — APPROVED** [review]
Looks good to me.

**Pull request merged**